app/lib/route_data_manager.rb

```diff
@@ -6,6 +6,8 @@ def initialize(trainee:)
   end
 
   def update_training_route!(route)
+    return if route == "other"
```

Comment:
```ruby
RouteDataManager.new(trainee: Trainee.first).update_training_route! "other"

# ArgumentError: 'other' is not a valid training_route
```

You don't really need this as the controller has guarded that

Reply:
thanks